src/components/Filter/OptionDisplay.module.css

```diff
@@ -7,29 +9,30 @@
 
 .optionList {
   display: flex;
-  align-items: left;
+  align-items: start;
   flex-direction: column;
   border-left: 1px solid var(--fds-semantic-border-neutral-divider);
   border-right: 1px solid var(--fds-semantic-border-neutral-divider);
   border-top: 1px solid var(--fds-semantic-border-neutral-divider);
 }
 
 .option {
+  width: 100%;
   border: none;
   border-bottom: 1px solid var(--fds-semantic-border-neutral-divider);
-  padding: 8px 10px 8px 10px;
   background-color: var(--fds-semantic-background-default);
   cursor: pointer;
   font: inherit;
   text-align: left;
+  padding: 0;
 }
 
 .option:hover {
-  background-color: var(--fds-semantic-surface-action-subtle);
+  background-color: var(--fds-semantic-surface-info-subtle);
```

Comment:
Honestly, because I couldn't get that color to work for some reason. It just didn't show up.
But this other token worked and at that point I was pretty tired of things not working so I just went for it instead 😅

Reply:
I see, I can try to fix this when this is merged in